app/controllers/rubcop_test_controller.rb

```diff
@@ -1,17 +1,18 @@
-class RubocopController  < ApplicationController
-	def initialize
-		print "this is so bad patern"
-	end
+class RubocopController < ApplicationController
+  def initialize
+    print 'this is so bad patern'
+  end
 
-	def print_fib(range)
-		range.each do |n|
-			p fib(n)
-		end
-	end
+  def print_fib(range)
+    range.each do |n|
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 'n' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)